help/include/set-up-your-account.md

```diff
@@ -10,7 +10,7 @@
   [edit your profile information](/help/edit-your-profile) to tell others
   about yourself.
 
-- [Review your display settings](/help/review-your-settings#review-your-display-settings).
+- [Review your display settings](/help/review-your-settings#review-your-preferences).
```

Comment:
I think you're missing the updated section with this commit; the needed changes are mistakenly with the `tests: Change settings.test.ts file.`, so make sure you rewrite your history so those are part of this commit.